dnf-plugins/py3query.py

```diff
@@ -268,7 +269,14 @@ def bug_namegetter(bug):
                 status = bug.status
                 if bug.resolution:
                     status += ' ' + bug.resolution
-                r.setdefault('links', {})['bug'] = [bug.weburl, status]
+                # Let's get the datetime of the last comment
+                last_comment_datetime = None
+                if bug.comments:
```

Comment:
This makes a new request to Bugzilla for every bug. That's quite slow and inefficient.
Could you add `last_change_time` to `include_fields` above, and use `bug.last_change_time`? It's "last activity" rather than "last commit", but it's much cheaper to get.